doc/en/example/assertion/test_failures.py

```diff
@@ -5,9 +5,9 @@
 pytest_plugins = ("pytester",)
 
 
-def test_failure_demo_fails_properly(testdir):
-    target = testdir.tmpdir.join(os.path.basename(failure_demo))
+def test_failure_demo_fails_properly(pytester):
```

Comment:
as a followup we might want to make this use examples

Reply:
This one I'm not exactly sure where it's used, so left it as is.